app/assets/javascripts/darkswarm/controllers/registration/registration_controller.js.coffee

```diff
@@ -1,8 +1,11 @@
-Darkswarm.controller "RegistrationCtrl", ($scope, RegistrationService, EnterpriseRegistrationService, availableCountries) ->
+Darkswarm.controller "RegistrationCtrl", ($scope, RegistrationService, EnterpriseRegistrationService, availableCountries, GmapsGeo) ->
```

Comment:
I think you need to make this file independent of the map provider. If gmaps is deactivated will this dependency  GmapsGeo be there? I'd recommend creating a proxy service called GeoCoder and than include GmapsGeo in it if google maps are activated in the instance, otherwise, fail gracefully.